python/tests/tornado/test_hyperbahn.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2015 Uber Technologies, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+import mock
+import pytest
+
+from tchannel.exceptions import ConnectionClosedException
+from tchannel.tornado.hyperbahn import HyperbahnClient
+
+
+def test_new_client_establishes_peers():
+    tchannel = mock.MagicMock()
+    routers = [mock.sentinel.foo, mock.sentinel.bar]
+
+    HyperbahnClient(
+        service='baz',
+        hyperbahn_routers=routers,
+        tchannel=tchannel,
+    )
+
+    for router in routers:
+        assert tchannel.peers.add.called_with(router)
```

Comment:
Instead of testing the behavior by asserting this call, can you instead assert that peers passed into the constructor exist? Something like,

    assert tchannel.peers.lookup('localhost:4040')

For all peers